ui/src/Components/AddCompany/AddCompany.js

```diff
@@ -1,34 +1,196 @@
+               
+                    
+                    <Link to="/" className="text-black text-decoration-none ms-2"> <BsHouseFill  className='me-2'/>Home</Link>/
+                    <Link to="/Company" className="text-black text-decoration-none">Company</Link>
+
+                </div>
+                <div>
+                    <b>Companies</b>
+                </div>
 
-function AddCompany(props){
-    return(
-        <div className="card m-3 p-4 w-75 m-auto mt-4">
-            <h4 className="m-auto">Add Company Information</h4>
-            <div className="d-flex mt-3">
-                <input type="text" placeholder="CompanyName" className="form-control" />
-                <input type="text" placeholder="MobileNumber" className="form-control ms-2" />
-            </div>
-            <div  className="d-flex mt-3">
-                <input type="text" placeholder="addressLine1" className="form-control" />
-                <input type="text" placeholder="addressLine2" className="form-control ms-2" />
-            </div>
-            <div  className="d-flex mt-3">
-                <input type="text" placeholder="createdDate" className="form-control" />
-                <input type="text" placeholder="landMark" className="form-control ms-2" />
-            </div>
-            <div  className="d-flex mt-3">
-                <input type="text" placeholder="logoName" className="form-control" />
-                <input type="text" placeholder="pinCode" className="form-control ms-2" />
-            </div>
-            <div  className="d-flex mt-3">
-                <input type="text" placeholder="state" className="form-control" />
-                <input type="text" placeholder="town" className="form-control ms-2" />
-            </div>
-            <div className="mt-4">
-                <button className="btn btn-primary">Save</button>
-                <button className="btn btn-danger ms-2" onClick={props.onCancel}>Cancel</button>
             </div>
+            <ReactTooltip />
+
+
+
+            <Formik initialValues={props.initialValues}
+                validationSchema={Yup.object({
+                    companyName: Yup.string().required('company name is required'),
+                    mobileNumber: Yup.string().required('phone number is required'),
+                    addressLine1: Yup.string().required('address is required'),
+                    logoName: Yup.string().required('logo name is required'),
+                    addressLine2: Yup.string().required('address  is required'),
+                    state: Yup.string().required('state is required'),
+                    town: Yup.string().required('town is required'),
+                    pinCode: Yup.string().required('pincode is required')
+
+                })}
+                onSubmit={(values) => {
+                    if(values && values.id){
+                        axios.put("https://trinitstechnologies.com/demo/api/v1/companies/"+values.id, values)
```

Comment:
Use the constant value for https://trinitstechnologies.com/demo/api/v1/companies/